README.md

```diff
@@ -4,39 +4,39 @@
 * Create and login customer users to build personalized experiences.
 
-CIF Cloud services are exposed via REST style APIs. All API access via Adobe I/O Runtime is over HTTPS. All CIF services return the information that you request as JSON data format.
+CIF REST services are exposed via REST style APIs. All API access via Adobe I/O Runtime is over HTTPS. All CIF REST services return the information that you request as JSON data format.
 
-The CIF Cloud API is not a full CRUD commerce admin or management API, for example it is not designed for data import / export use cases.
+The CIF REST API is not a full CRUD commerce admin or management API, for example it is not designed for data import / export use cases.
 
 ### API Specification
 
 The API and data models are specified in [swagger.yaml](src/main/resources/swagger/swagger.yaml) / [swagger.json](src/main/resources/swagger/swagger.json). Review directly in [Swagger UI](http://opensource.adobe.com/commerce-cif-api/).
 
 The API and models are actually defined in Java with both `swagger` and `jax-rs` annotations, so we do not directly edit the Swagger JSON and YAML specification. These Java definitions can be found under [src/main/java](src/main/java). They are used to automatically generate:
 * the Swagger specification: [src/main/resources/swagger](src/main/resources/swagger)
-* the Javascript model classes used by the CIF implementations written in `Node.js`: [src/main/resources/javascript](src/main/resources/javascript)
+* the Javascript model classes used by the CIF REST implementations written in `Node.js`: [src/main/resources/javascript](src/main/resources/javascript)
 * the Nginx configuration that can be used to configure the CIF REST API endpoints on an Openwhisk environment: [src/main/resources/nginx](src/main/resources/nginx)
 
 Note that the build actually generates these files under the `src/main/resources/generated` folder, but this folder is not committed on the git repository. The `maven release` process actually copies the generated files to the folders mentioned above so that we only have released files in the repository.
 
 ### GraphQL endpoint and specification
 
-Our Swagger API also defines the `/graphql` HTTP endpoint for the CIF [GraphQL](https://graphql.org/) API. Because Swagger is only designed to document a REST API, the GraphQL API and schema has its own [documentation](http://opensource.adobe.com/commerce-cif-api/graphql/index.html). Check this documentation to find out about the operations currently supported by the CIF GraphQL API.
+Our Swagger API also defines the `/graphql` HTTP endpoint for the CIF REST [GraphQL](https://graphql.org/) API. Because Swagger is only designed to document a REST API, the GraphQL API and schema has its own [documentation](http://opensource.adobe.com/commerce-cif-api/graphql/index.html). Check this documentation to find out about the operations currently supported by the CIF REST GraphQL API.
```

Comment:
I guess here having REST with GraphQL is totally misleading, but fine if we anyway deprecate the GraphQL part and remove it from the documentation.

Reply:
This section should be remove anyway, we have CIF-737 for that.